src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardController.js

```diff
@@ -1,118 +1,62 @@
 ({
-
-    /******************************** Batch Details Controller Functions *****************************/
-
-    /**
-     * @description Constructs the Model Module, and the View Modules.
-     */
-    doInit: function (component, event, helper) {
-        var bgeBatchController = helper.BGEBatchController(component);
-
-        var model = helper.DetailsModel();
-        model.setBackendController(bgeBatchController);
-        component.set('v.model', model);
-
-        var batchInfoView = helper.BatchInfoView(component, model);
-        component.set('v.batchInfo', batchInfoView);
-        var batchMetadataView = helper.BatchMetadataView(component, model);
-        component.set('v.batchMetadata', batchMetadataView);
-        var batchFieldsView = helper.BatchFieldsView(component, model);
-        component.set('v.availableFields', batchFieldsView);
-        var batchFieldOptionsView = helper.BatchFieldOptionsView(component, model);
-        component.set('v.batchFieldOptions', batchFieldOptionsView);
-
-        model.init(component);
-    },
-
-    /**
-     * @description Navigates from View to Edit mode.
-     */
-    changeModeToEdit: function(component, event, helper) {
-        var model = component.get('v.model');
-        model.getBatchMetadata().setMode('edit');
+    doInit: function(component, event, helper){
+        helper.init(component);
     },
 
     /**
-     * @description handles ltng:sendMessage
+     * @description handles incoming ltng:sendMessage from footer
      * must be explicit about channel because other messages may be sent
      */
     handleSendMessage: function(component, event, helper) {
         const task = event.getParam('channel');
-        let model = component.get('v.model');
 
-        // handle cancel
-        if (task === 'cancel') {
```

Comment:
Just noting here that cancel is handled directly by the generic footer component, so we don't need to handle it here now that we are always in modal context with footer.